QGL/BasicSequences/Feedback.py

```diff
@@ -46,24 +56,28 @@ def Reset(qubits, measDelay = 1e-6, signVec = None, doubleRound = True, buf = 20
 	-------
 	plotHandle : handle to plot window to prevent destruction
 	"""
-	if measChans is None:
-		measChans = qubits
+    if measChans is None:
+        measChans = qubits
 
-	if signVec == None:
-		signVec = (0,)*len(qubits)
+    if signVec == None:
+        signVec = (0, ) * len(qubits)
```

Comment:
This is an odd change. Is there an option to make empty tuples not have an extra space?